src/components/layout/_dom.js

```diff
@@ -19,7 +19,7 @@ const Dom = ({ dom }) => {
       <h1 className='absolute w-full text-xs tracking-wider text-center text-white text-gray-100 md:mt-56 mt-28 top-1/2 sm:subpixel-antialiased md:antialiased'>
         REACT THREE NEXT STARTER
       </h1>
-      <div className='absolute bottom-4 right-4 z-index-30'>
+      <div className='absolute p-2 m-2 right-4 z-index-30'>
```

Comment:
The pmndrs badge was hidden outside of the view ( top right ) this place it on the top right aligned with the top left link.